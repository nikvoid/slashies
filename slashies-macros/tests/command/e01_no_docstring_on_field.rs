use slashies_macros::Command;

/// A command with an invalid field that has no description
#[derive(Command)]
#[name = "BadCommand"]
struct BadCommand {
    bad_field: u64,
    /// Ok field
    good_field: u64,
}

fn main() {}
