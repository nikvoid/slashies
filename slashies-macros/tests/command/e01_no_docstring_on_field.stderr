error: Command options must specify a description via a docstring
 --> tests/command/e01_no_docstring_on_field.rs:7:5
  |
7 |     bad_field: u64,
  |     ^^^^^^^^^
